framework/tlvf/AutoGenerated/src/tlvf/test/tlvVarList.cpp

```diff
@@ -105,9 +105,12 @@ std::shared_ptr<cInner> tlvTestVarList::create_complex_list() {
     }
     m_lock_order_counter__ = 1;
     m_lock_allocation__ = true;
-    uint8_t *src = (uint8_t *)m_complex_list;
-    uint8_t *dst = (uint8_t *)m_complex_list + len;
+    uint8_t *src = (uint8_t*) m_complex_list;
```

Comment:
Here also - `src` is set twice

Reply:
That's intended.
The first assignment is setting a default value (the beginning of the list), for the case of an empty list (in which the code won't enter the if and won't execute line 111).